docs/src/_includes/components/npx_tabs.macro.html

```diff
@@ -0,0 +1,46 @@
+{%- macro npx_tabs(params) -%}
+<div class="c-tabs" data-tabs>
+   <div class="c-tabs__tablist" data-tablist hidden>
+      <button class="c-tabs__tab" data-tab>npm</button>
+      <button class="c-tabs__tab" data-tab>yarn</button>
+      <button class="c-tabs__tab" data-tab>pnpm</button>
+      <button class="c-tabs__tab" data-tab>bun</button>
+   </div>
+   <div class="c-tabs__tabpanel" data-tabpanel>
+      <h3 class="h6 c-tabs__tabpanel__title">npm</h3>
+
+```shell
+{% if params.comment %}# {{ params.comment | safe }}{{ "\n" }}{% endif -%}
+{% if params.previousCommands %}{% for item in params.previousCommands %}{{ item | safe }} {% endfor %}| {% endif -%}npx {{ params.package | safe }} {% for item in params.args %}{{ item | safe }} {% endfor %}
```

Comment:
```suggestion
{% if params.previousCommands %}{% for item in params.previousCommands %}{{ item | safe }} {% endfor %}| {% endif -%}npx {{ params.package | safe }} {% for item in params.args %}{{ item | safe }}{% endfor %}
```

we can avoid extra space in the end.
![Screenshot 2024-09-26 151658](https://github.com/user-attachments/assets/f7fac396-b1f2-4a57-9605-d91e49f8cbb0)